bcipy/gui/viewer/data_viewer.py

```diff
@@ -64,6 +59,11 @@ def __init__(self,
         # Default filter
         self.filter = Downsample(self.downsample_factor)
 
+        self.refresh_rate = refresh
```

Comment:
@lawhead I addressed all comments I intended in this PR. I added this after testing everything and running into an issue with downsample in the data viewer. Can you look this part over to make sure it looks ok?

Changed:

- Output of Downsample when fs is not defined
- samples per second uses the downsample rate 
- changed default (not sure how this impacts everything though)

Reply:
It seems like the samples_per_second is not correct. This value is used to get the raw data before the filters are applied so it doesn't seem like it would retrieve enough data.